CIP-XXXX/README.md

```diff
@@ -0,0 +1,60 @@
+---
+CIP: ?
+Title: CIP-30 ext: Bulk transaction signing
```

Comment:
@Ryun1 what would you propose as a naming convention for these extensions?  We're bound to get a lot of them and I think it would be helpful if they were identifiable in a list of CIP titles... without sticking out too much 🤔

Reply:
See issue #590.

Reply:
Changed according to https://github.com/cardano-foundation/CIPs/issues/590 but will leave it unresolved until suggestion is final.

Reply:
Looks good so far & is in agreement with all those who have expressed an opinion 🤓